### PR TITLE
GEOPY-244

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-244 - Update write_ui_json to check for uuid strings and write to property field instead of value field**

### CODE REVIEW

File: geoapps/io/params.py [hunk 1]
```diff
@@ -214,8 +214,11 @@ def to_dict(self, ui_json: dict = None, ui_json_format=True):
                 if isinstance(ui_json[k], dict):
                     field = "value"
                     if "isValue" in ui_json[k].keys():
-                        if ui_json[k]["isValue"] is False:
+                        if isinstance(new_val, UUID):
+                            ui_json[k]["isValue"] = False
```
Comment (benk-mira) [inline geoapps/io/params.py:218]: Is this correct in the case of a parent object?  We, or possibly just I, have been using value to store the UUID of objects to which data 'channels' are added.  Eg: topography_object -> value: some_uuid, topography -> property: some_uuid, value: some_float.

Comment (domfournier) [reply geoapps/io/params.py:218]: yea but the parent won't have `isValue`  in the dict, so we know that only applies for the channel field?

Comment (benk-mira) [reply geoapps/io/params.py:218]: yep, you're right!

**Review by benk-mira — APPROVED** [review]
Thanks for the updates and tests!

**Review by benk-mira — APPROVED** [review]

**Pull request merged**